lib/jazzy/config.rb

```diff
@@ -127,6 +127,12 @@ def expand_path(path)
       description: 'The SDK for which your code should be built.',
       default: 'macosx'
 
+    config_attr :hide_declarations,
+      command_line: '--hide-declarations [swift|objc] ',
+      description: 'Hide language declarations when generating Objective-C '\
+                   'framework.',
+      default: ''
+
```

Comment:
Be good to police the user enters one of these strings + nothing else, and also that `--objc` is set -- I know jazzy doesn't do much of that today but that does cause user confusion.  (also wording, "...generating *docs for an* Objective...")

Reply:
updated the description. Those are all good to have, but I don't see any check at all on config today. Maybe I can do a separated PR to do it once for all. I also only check the existence of 'swift or objc' in many places and fix others too, so putting a any other string will have no effect. It'll simply ignore.